sonar-text-plugin/src/test/resources/secretsConfiguration/postFilter/postFilterSpec.yaml

```diff
@@ -0,0 +1,50 @@
+provider:  
+  metadata:
+      name: provider name
+      category: Cloud provider
+      references:
+        - description: Reference 1
+          link: https://docs.aws.amazon.com/IAM/...
+          type: Standards
+        - description: Reference 2
+          link: https://docs.aws.amazon.com/IAM/...
+          type: Documentation
+        - description: Reference 3
+          link: https://docs.aws.amazon.com/IAM/...
+          type: Conference Presentations
+        - description: Reference 4
+          link: https://docs.aws.amazon.com/IAM/...
+          type: Articles & blog posts
+      message: provider message
+      fix: provider fix
+      impact: provider impact
+  detection:
+    matching:
+      pattern: "\\b(provider matching pattern)\\b"
+  rules:
+    - id: exampleId
+      rspecKey: exampleKey
+      metadata:
+        name: rule name
+        defaultProfile: false
+        charset: "[0-9a-z\\/+]"
+        message: rule message
+        impact: rule impact
+        fix: rule fix
+        references:
+          - description: rule reference
+            link: https://docs.aws.amazon.com/IAM/...
+            type: Standards
+      detection:
+        matching:
+          pattern: "\\b(rule matching pattern)\\b"
+        post:
+          statisticalFilter:
+            threshold: 4.2
+          patternNot: EXAMPLEKEY
```

Comment:
Maybe add other fields here as well (`input_string`, `heuristics`) as they are already present in the model?